Updates to topics for MW UI changes. (#427)
Concordium/concordium.github.io: source/shared/net/mobile-wallet/make-simple-transfer-mw.rst

Make a CCD transfer from an account
===================================

> #. Go to the Accounts page and tap **SEND** on the account from which you want to send CCD.

Reviewer: Would it make sense to have a screenshot of the Accounts screen here?

#. Go to the Accounts page.

#. Tap **SEND** on the account from which you want to send CCD.

#. Enter the amount you want to transfer.

> #. In **Paste recipient address** select the recipient of the transfer.

Reviewer: ```suggestion
#. In **Paste recipient address** you can paste the recipient address.
```

>    - You can either choose a recipient that already exists in your **Address Book**, or tap **Scan QR** to add a new one.

Reviewer: ```suggestion
   - You can also choose a recipient that already exists in your **Address Book**, or tap **Scan QR** to add a new one.
```
